Userland/Libraries/LibJS/Runtime/TypedArray.cpp

```diff
@@ -405,5 +416,4 @@ void TypedArrayBase::visit_edges(Visitor& visitor)
     JS_DEFINE_TYPED_ARRAY(ClassName, snake_name, PrototypeName, ConstructorName, ArrayType);
 JS_ENUMERATE_TYPED_ARRAYS
 #undef __JS_ENUMERATE
-
```

Comment:
We usually keep the whitespace between the closing namespace curly and the inner code, no?

Reply:
Definitely, clang-format does this when you format while having one too many curly braces somewhere, and doesn't add it back later. Fixed.